### ISSUE TITLE
Support for PPG in azurerm_kubernetes_cluster_node_pool and azurerm_kubernetes_cluster default node pool

### ISSUE THREAD

**Issue by james-knott** [hashicorp/terraform-provider-azurerm#7550]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I see this is available now. Any idea if or when this will be added?

https://docs.microsoft.com/en-us/azure/aks/reduce-latency-ppg

Thank you

### New or Affected Resource(s)

* azurerm_kubernetes_cluster

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by KristapsT** [comment]
This appears now to be possible for azurerm_kubernetes_cluster_node_pool, but still not with azurerm_kubernetes_cluster default node pool.

It is also not possible through Azure UI when creating a new AKS cluster, is this Azure API limitation where it doesn't allow for placement group assessment when creating a new cluster?

**Comment by stephybun** [comment]
Support for this added by #9164 and #9195 and was released in **v2.36.0** of the provider.
